Cabal/doc/developing-packages.markdown

```diff
@@ -2071,6 +2071,13 @@ getLibexecDir :: IO FilePath
 getSysconfDir :: IO FilePath
 ~~~~~~~~~~~~~~~
 
+The actual location of all these directories can be individually
+overridden at runtime using environment variables of the form
+`pkg_name_var`, where `pkg_name` is the name of the package with
+all hyphens converted into underscores, and `var` is either
+`bindir`, `libdir`, `datadir`, `libexedir` or `sysconfdir`.
```

Comment:
It may be useful to give a concrete example for some well-known package name as to how such a var looks like for demonstration purposes